src/pages/account/center/service.ts

```diff
@@ -0,0 +1,11 @@
+import request from '@/utils/request';
+
+export async function queryCurrent() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'queryCurrent' to have a typedef](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)